ansible/files/walg_helper_scripts/wal_change_ownership.sh

```diff
@@ -0,0 +1,42 @@
+#! /usr/bin/env bash
+
+set -euo pipefail
+
+string=$1
```

Comment:
can we come up with a better name for this than $string?

Reply:
Have updated it to `$filename` instead.